src/main/interface/settings.c

```diff
@@ -944,6 +944,7 @@ const clivalue_t valueTable[] = {
 #ifdef USE_SDCARD
     { "sdcard_detect_inverted",     VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_SDCARD_CONFIG, offsetof(sdcardConfig_t, cardDetectInverted) },
     { "sdcard_mode",                VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_SDCARD_MODE }, PG_SDCARD_CONFIG, offsetof(sdcardConfig_t, mode) },
+    { "sdcard_enabled",             VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_SDCARD_CONFIG, offsetof(sdcardConfig_t, enabled) },
```

Comment:
Hmm, looking into this, I am wondering if we still need this parameter, considering `sdcard_mode` has an `OFF` setting. We probably should just remove it as part of the refactoring, and use only `mode`.